### PR TITLE
Update d2l-input-textarea to use the Lit implementation from core.

### PR DESCRIPTION
**Pull request by dbatiste**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +25,7 @@
     "sauce-connect-launcher": "^1",
```
Comment (svanherk) [inline package.json:25]: If there's no more tests, I think you can cleanup some of the dependencies here and the `wct.conf.json` file

File: package.json [hunk 2]
```diff
@@ -6,12 +6,8 @@
   },
   "name": "d2l-inputs",
   "scripts": {
-    "lint": "npm run lint:wc && npm run lint:js",
-    "lint:js": "eslint . --ext .js,.html test/**/*.html demo/**/*.html",
-    "lint:wc": "polymer lint",
```
Comment (svanherk) [inline package.json:11]: Is it still worth linting the remaining Polymer style files?

File: package.json (hunk 1 above)
Comment (dbatiste) [reply package.json:25]: Right... thanks.  I'll clean this up a bit more.  😄

File: package.json (hunk 2 above)
Comment (dbatiste) [reply package.json:11]: Yeah, I was probably a bit hasty.  😏

**Comment by svanherk** [comment]
Is it worth adding some simple GitHub action workflows here?  Probably not, I expect we're archiving this after you merge?

**Comment by dbatiste** [comment]
> Is it worth adding some simple GitHub action workflows here? Probably not, I expect we're archiving this after you merge?

I was on the fence about that, but yeah, it's going to get archived again after merging.

**Comment by svanherk** [comment]
>Is it worth adding some simple GitHub action workflows here? Probably not, I expect we're archiving this after you merge?
>> I was on the fence about that, but yeah, it's going to get archived again after merging.

I'm good with adding it later, if we ever need to

**Review by svanherk — APPROVED** [review]

**Review by dlockhart — APPROVED** [review]

**Pull request merged**